src/zenml/models/v2/core/pipeline_run.py

```diff
@@ -237,6 +237,9 @@ class PipelineRunResponseMetadata(WorkspaceScopedResponseMetadata):
         default=False,
         description="Whether a template can be created from this run.",
     )
+    steps_substitutions: Dict[str, Dict[str, str]] = Field(
+        title="Substitutions used in the step runs of this pipeline run.",
```

Comment:
If we give this a default value, we might avoid some client-server mismatch errors? Do you think it messes with something if we do that?

Reply:
Nope, shall be good to go, added the `default_factory`